pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -452,10 +452,6 @@ func (c *ReverseExpandQuery) reverseExpandDirect(
 		})
 	}
 
-	sourceRelationRef := &openfgav1.RelationReference{
```

Comment:
these lines that were removed got picked up by govet?

Reply:
Yes :)